std/regex/package.d

```diff
@@ -1248,18 +1235,21 @@ public @trusted void replaceAllInto(alias fun, Sink, R, RegEx)
     replaceMatchesInto!fun(sink, input, matchAll(input, re));
 }
 
-// a bit of examples
+///
 @system unittest
 {
-    //swap all 3 letter words and bring it back
-    string text = "How are you doing?";
-    auto sink = appender!(char[])();
-    replaceAllInto!(cap => retro(cap[0]))(sink, text, regex(`\b\w{3}\b`));
-    auto swapped = sink.data.dup; // make a copy explicitly
-    assert(swapped == "woH era uoy doing?");
-    sink.clear();
-    replaceAllInto!(cap => retro(cap[0]))(sink, swapped, regex(`\b\w{3}\b`));
-    assert(sink.data == text);
+    // insert comma as thousands delimiter in fifty randomly produced big numbers
+    import std.array, std.random, std.conv, std.range;
+    static re = ctRegex!(`(?<=\d)(?=(\d\d\d)+\b)`, "g");
```

Comment:
Is there any reason to use `ctRegex` instead of just plain `regex`?

IIRC, `ctRegex` has recently found to have not as much performance benefit as originally thought, but a pretty high compile-time cost. For a documented unittest, it seems a bit overkill to use `ctRegex`.